packages/@okta/vuepress-site/docs/guides/build-api-integration/main/index.md

```diff
@@ -0,0 +1,260 @@
+---
+title: Build an API service integration
+excerpt: Learn how to build, test, and register an API service integration to the Okta Integration Network.
+layout: Guides
+---
+
+<ApiLifecycle access="ea" /><!--For both Classic and Identity Engine-->
+
+This guide explains how to build, test, and submit an API service integration to the Okta Integration Network (OIN) for review and publishing.
+
+---
+
+**Learning outcomes**
+
+* Learn how to build, test, and submit an API service integration that can be published in the OIN.
+
+**What you need**
+
+* [Okta Developer Edition organization](https://developer.okta.com/signup)
+   > **Note:** The API Service Integration is a [Self-service Early Access (EA)](/docs/reference/releases-at-okta/#early-access-ea) feature. Enable this feature in your org to obtain the API service integration capability.
+* A service app that needs to access Okta APIs for your customer
+
+**Sample code**
+
+* See [How to use Client Credentials flow with Spring Security](https://developer.okta.com/blog/2021/05/05/client-credentials-spring-security) blog for an example of a Client Credentials flow using the [Okta Spring Boot Starter](https://github.com/okta/okta-spring-boot) library.
+
+---
+
+## Overview
+
+A service-to-service app where where a backend service or a daemon calls [Okta management APIs](/docs/reference/core-okta-api/) for a tenant (Okta org) can be published in the Okta Integration Network (OIN) as an API service integration.
```

Comment:
Typo in `where where`

Reply:
Note: Please check if `daemon calls` is correct grammatically, as `backend service or a daemon` may have been intended to be a noun in the sentence.

Reply:
Thanks! I'll make the correction